tests/integration/test_charm.py

```diff
@@ -127,6 +129,7 @@ async def test_consistent_data_replication_across_cluster(ops_test: OpsTest) ->
 
 @pytest.mark.order(3)
 @pytest.mark.abort_on_fail
+@pytest.mark.ha_tests
```

Comment:
The tests for charms are going to grow quite large, it might be useful to split things into separate files instead of using `@pytest.mark.ha_tests`. Check out [this WIP](https://github.com/canonical/mongodb-operator/pull/61) PR that does this. But if you prefer your way of doing this with `@pytest.mark.ha_tests` then feel free to keep as is :)

Reply:
To clarify, they are mostly in segregated files already.
This is used in the tox environment for each test, so it skips all other `marks`, avoiding the necessity for writing the test file path.